### PR TITLE
Topic/u325 007

### PR DESCRIPTION
**Pull request by AnthonyLeonardoGracio**

### CODE REVIEW

File: testsuite/ada_lsp/SA24-055.called_by_on_abstract/test.json [hunk 1]
```diff
@@ -14,45 +14,113 @@
...
+                     "implementationProvider": true,
+                     "referencesProvider": true,
+                     "documentHighlightProvider": true,
+                     "documentSymbolProvider": {},
+                     "codeActionProvider": {},
+                     "documentFormattingProvider": true,
+                     "renameProvider": {},
+                     "foldingRangeProvider": true,
+                     "executeCommandProvider": {
+                        "commands": [
```
Comment (joaopsazevedo) [inline testsuite/ada_lsp/SA24-055.called_by_on_abstract/test.json:102]: I would remove this for two reasons: 1) they're not relevant for what we're testing and 2) every time we add a new command, we'll need to change these tests even if they don't have anything to do with the new command.

**Review by joaopsazevedo — APPROVED** [review]
Good PR, just two minor comments, but feel free to push.

File: testsuite/ada_lsp/SA24-055.called_by_on_abstract/test.json (hunk 1 above)
Comment (AnthonyLeonardoGracio) [reply testsuite/ada_lsp/SA24-055.called_by_on_abstract/test.json:102]: Ok I'll remove it then

File: source/ada/lsp-ada_handlers.adb [hunk 2]
```diff
@@ -2675,42 +2632,46 @@ package body LSP.Ada_Handlers is
...
+                (C.all, Definition, This_Imprecise);
+         begin
+            Imprecise := Imprecise or This_Imprecise;
+
+            Calls_Cursor := Calls.First;
+            while Laltools.Common.References_By_Subprogram.Has_Element
+              (Calls_Cursor)
+            loop
+               declare
+                  Node          : constant Defining_Name :=
```
[the remark this answers is not in the document]
Comment (AnthonyLeonardoGracio) [reply source/ada/lsp-ada_handlers.adb:2648]: This come will be removed after anyway ;)

**Pull request merged**